dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamic/simulation/tools/RunDynamicSimulationTool.java

```diff
@@ -0,0 +1,204 @@
+
+        context.getOutputStream().println("Loading network '" + caseFile + "'");
+        Properties inputParams = readProperties(line, ConversionToolUtils.OptionType.IMPORT, context);
+        ImportConfig importConfig = (!skipPostProc) ? ImportConfig.load() : new ImportConfig();
+        Network network = Importers.loadNetwork(caseFile, context.getShortTimeExecutionComputationManager(),
+            importConfig, inputParams);
+        if (network == null) {
+            throw new PowsyblException("Case '" + caseFile + "' not found");
+        }
+
+        DynamicSimulationParameters params = DynamicSimulationParameters.load();
+        DynamicSimulationResult result = DynamicSimulation.run(network,
+            context.getShortTimeExecutionComputationManager(), params);
+
+        if (outputFile != null) {
+            exportResult(result, context, outputFile, format);
+        } else {
+            printResult(result, context);
+        }
+    }
+
+    private void printDynamicSimulationResult(DynamicSimulationResult result, Path outputFile,
+        TableFormatterFactory formatterFactory,
+        TableFormatterConfig formatterConfig) {
+        try (Writer writer = Files.newBufferedWriter(outputFile, StandardCharsets.UTF_8)) {
+            printDynamicSimulationResult(result, writer, formatterFactory, formatterConfig);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    private void printDynamicSimulationResult(DynamicSimulationResult result, Writer writer,
+        TableFormatterFactory formatterFactory,
+        TableFormatterConfig formatterConfig) {
+        try (TableFormatter formatter = formatterFactory.create(writer,
+            "dynamic simulation results",
+            formatterConfig,
+            new Column("Result"),
+            new Column("Metrics"))) {
+            formatter.writeCell(result.isOk());
+            formatter.writeCell(result.getMetrics().toString());
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    private void printResult(DynamicSimulationResult result, ToolRunningContext context) {
+        Writer writer = new OutputStreamWriter(context.getOutputStream());
+
+        AsciiTableFormatterFactory asciiTableFormatterFactory = new AsciiTableFormatterFactory();
+        printDynamicSimulationResult(result, writer, asciiTableFormatterFactory, TableFormatterConfig.load());
+    }
+
+    private void exportResult(DynamicSimulationResult result, ToolRunningContext context, Path outputFile,
+        Format format) {
+        context.getOutputStream().println("Writing results to '" + outputFile + "'");
+        switch (format) {
+            case CSV:
+                CsvTableFormatterFactory csvTableFormatterFactory = new CsvTableFormatterFactory();
+                printDynamicSimulationResult(result, outputFile, csvTableFormatterFactory, TableFormatterConfig.load());
+                break;
```

Comment:
It's easy to print a table here... but when the result will be more complex, it will be very difficult to export result to a CSV file